stylesheets/signup/signup.css

```diff
@@ -0,0 +1,52 @@
+@import url(/stylesheets/logIN/login.css);
+.sign-up-page-container 
+{
+    min-height: 80vh;
+    max-width:80%;
+    display:flex;
+    flex-direction: column;
+    box-shadow: 2px 2px 5px var(--dark-stone);
+    align-items: center;
+    margin: auto;
+}
+.text-size-xl
```

Comment:
one suggestion after your completion or before completion make typography seprated.  so you can reuse in other pages also

Reply:
i am already reusing the text-utilities classes from CL